CHANGES.md

```diff
@@ -1,6 +1,417 @@
 Release Changes List
 ====================
 
+*** stopped at dced642 2017-09-24 ***
+
+TODO: move docs to top-level? ('latest' is symbolic link to '.'?)
+TODO: remove 'master'/'latest' from any docs links?
+TODO: where to put LLVM codegen flags in 6135 / 8d704f0
+TODO: error-handling keyword highlighting in emacs
+TODO: 'tics' vs. `backtics`?
+TODO: refactor / filter highlights
+TODO: reorder sections
+
+version 1.16.0
+==============
+
+Nineteenth public release of Chapel, September 20, 2017
+
+Highlights (see subsequent sections for further details)
+--------------------------------------------------------
+* added a new 'configure'/'make install' option for building+installing Chapel
+* added prototypical support for 'Mason' v0.1.0, Chapel's package manager
+* added a new c2chapel tool that converts C headers to Chapel extern decls
+* 'chpl'-generated executables now take the main module name rather than 'a.out'
+* added a 'defer' statement to aid with cleanup
+* added support for 'try'/'try!' expressions (no 'catch' permitted)
+* added a 'prototype' keyword to relax checks for explicit module declarations
+* significantly improved the implementation of initializers
+* significantly improved support for error-handling
+* significantly improved the generality and robustness of 'forwarding' fields
+* generalized LayoutCSR to LayoutCS to support CSR and CSC sparse layouts
+* made the ReplicatedDist distribution less odd and renamed it to 'Replicated'
+* added support for a new 'Crypto' module based on OpenSSL
+* added a new 'Collections' module  (TODO: also DistributedBag/Dequeue?)
+* added support for a new 'TOML' module (not 100% feature complete)
+* added support for a distributed guided and dynamic iterators
+* improved support for the MPI module when ugni, gasnet/aries, and/or qthreads
```

Comment:
'with' instead of 'when'?   (I debated with myself whether this was below the typo/spell/format bar and decided it was a change in meaning, if slight, and not something you'd do yourself later.)